lib/paratii.eth.events.js

```diff
@@ -0,0 +1,75 @@
+export class ParatiiEthEvents {
+  constructor (context) {
+    this._subscriptions = {}
+    this.subscribe = context.web3.eth.subscribe
+    this._topics = {
+      transfer: ['0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef']
+    }
+  }
+
+  addListener (eventType, options, listener) {
+    let subscription = null
+
+    switch (eventType) {
+      case 'newBlockHeaders':
+      case 'syncing':
+      case 'pendingTransactions':
+        subscription = this.subscribe(eventType, listener)
+        break
+      case 'logs':
+        if (options === undefined) {
+          options = {
+            fromBlock: null,
+            topics: null
+          }
+        }
+        subscription = this.subscribe('logs', options, listener)
+        break
+      default:
+        if (options === undefined) {
+          options = {
+            fromBlock: null,
+            topics: this._topics[eventType]
+          }
+        }
+        subscription = this.subscribe('logs', options, listener)
```

Comment:
We want to subscribe to the logs of a specific contract. I.e. to the `Transfer` event of the contract that is on `paratii.eth.getContractAddress('ParatiiToken')`. I think that works by settings `options.address` to this address : http://web3js.readthedocs.io/en/1.0/web3-eth-subscribe.html#id16

Reply:
it is not enough, we want to subscribe to particular events inside a contract.

Reply:
We want to do _both_. So in **addition** to what you are doing here (subscribing to a particular topic), we **also** want to specify that this Event must have logged by a certain specific contact instance. (I.e. we do not want all Transfer events, just those that come from our ParatiiToken)